src/app/api/kamu.graphql.interface.ts

```diff
@@ -2903,6 +2960,38 @@ export class RenameDatasetGQL extends Apollo.Mutation<RenameDatasetMutation, Ren
         super(apollo);
     }
 }
+export const SourceRootDatasetByAccountAndDatasetNameDocument = gql`
```

Comment:
I don't understand where this came from. There is no such document between your GraphQL sources.

Reply:
Removed, the old piece of code